scripts/postinstall.js

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
+const cp = require('child_process');
+
+function isGlobalNpmInstall() {
+  return !!process.env.npm_config_global;
+}
+
+function isGlobalYarnInstall() {
+  if (!process.env.npm_config_argv) return false;
+
+  const argv = process.env.npm_config_argv || { original: [] };
+  const parsed = JSON.parse(argv);
+  return parsed.original.includes('global');
```

Comment:
Yarn supports `-g` too

Reply:
@peternhale is that true? When I tested locally, `yarn add sfdx-cli --global` it installed into the local node_modules

Also, I can't find anything in the yarn docs about `--global` support